File: miband-sdk/src/main/java/com/khmelenko/lab/miband/listeners/RealtimeStepsNotifyListener.java
package com.khmelenko.lab.miband.listeners;

/**
 * Listener for realtime steps notifications
 *
 * @author Dmytro Khmelenko
 */
public interface RealtimeStepsNotifyListener {

    /**
     * Called when new notification arrived
     *
     * @param steps Steps amount
     */
    void onNotify(int steps);
}

File: miband-sdk-kotlin/src/main/java/com/khmelenko/lab/miband/listeners/RealtimeStepsNotifyListener.kt
package com.khmelenko.lab.miband.listeners

/**
 * Listener for realtime steps notifications
 *
 * @author Dmytro Khmelenko
 */
interface RealtimeStepsNotifyListener {

    /**
     * Called when new notification arrived
     *
     * @param steps Steps amount
     */
    fun onNotify(steps: Int)
}
